cfgov/jinja2/v1/about-us/the-bureau/about-deputy-director/index.html

```diff
@@ -19,12 +19,16 @@
 
 	{% import 'molecules/social-media.html' as social_media %}
 
-    <div class="block block__flush-top" data-qa-hook="director-bio">
+    <div class="block block__flush-top"
+         data-qa-hook="director-bio">
         <h1 data-qa-hook="director-bio-title">
             Deputy Director David Silberman (Acting)
         </h1>
         <p class="lead-paragraph">
-            David Silberman currently serves as the Acting Deputy Director as well as the Associate Director of Research, Markets, and Regulations at the Consumer Financial Protection Bureau (CFPB).
+            David Silberman currently serves as the acting deputy director
```

Comment:
Not being a content person, I offer my opinion that yes, I think so. So the first sentence is the lead and then the body text continues from there -- that matches what should happen on the director's bio too.